common/setups/rasr/util/nn.py

```diff
@@ -152,23 +152,27 @@ class OggZipHdfDataInput:
     def __init__(
         self,
         oggzip_files: List[tk.Path],
-        alignments: tk.Path,
+        alignments: List[tk.Path],
         context_window: Dict,
         audio: Dict,
         targets: Optional[str] = None,
         partition_epoch: int = 1,
         seq_ordering: str = "laplace:.1000",
         ogg_args: Optional[Dict[str, Any]] = None,
+        meta_args: Optional[Dict[str, Any]] = None,
         acoustic_mixtures: Optional[tk.Path] = None,
```

Comment:
Is this even used?

Reply:
Not directly here, but the system classes use it for e.g. recognition. It's also included in `ReturnnRasrDataInput`, so I added it here to match that behavior.